Src/Microsoft.Dynamic/zlib.net/Inflate.cs

```diff
@@ -259,12 +259,12 @@ internal int inflateInit(ZStream z, int windowBits)
 		internal int inflate(ZStream z, FlushStrategy flush)
 		{
 		    int internalFlush = (int)flush;
-
-            int res_temp;
 			
-			if (z == null || z.istate == null || z.next_in == null)
+			if (z?.istate == null || z.next_in == null)
```

Comment:
I'd prefer keeping the null check separate as it's clearer that it's also not null in the second condition.